tests/Unit/Domain/CoordinateMaps/Test_Wedge2D.cpp

```diff
@@ -101,9 +101,17 @@ SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge2D", "[Domain][Unit]") {
   CAPTURE_PRECISE(outer_radius);
   const int direction_number = int_dis(gen);
   CAPTURE_PRECISE(direction_number);
-  const Direction<2> direction{
-      static_cast<size_t>(direction_number > 1),
-      direction_number % 2 == 1 ? Side::Upper : Side::Lower};
+  const Direction<2> direction = [&direction_number]() {
```

Comment:
No random here.
`for (const auto& direction : Direction<2>::all_directions()) {`